libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -151,14 +151,14 @@ export async function buildCastVoteRecordReport({
       });
     }
 
-    const validatedSheet = validationResult.ok();
-    const [frontFilename, backFilename] = validatedSheet.wasReversed
+    const canonicalizedSheet = validationResult.ok();
+    const [frontFilename, backFilename] = canonicalizedSheet.wasReversed
       ? [sideTwoFilename, sideOneFilename]
       : [sideOneFilename, sideTwoFilename];
```

Comment:
Could consider passing in the file names with the page interpretations to `canonicalizeSheet` in order to avoid having to return the `wasReversed` flag

Reply:
will consider